.jshintrc

```diff
@@ -0,0 +1,30 @@
+{
+  "nonew": true,
+  "jquery": false,
+  "curly": true,
+  "latedef": true,
+  "unused": true,
+  "noarg": true,
+  "indent": 1,
+  "trailing": true,
+  "white": true,
+  "forin": true,
+  "noempty": true,
+  "quotmark": "single",
+  "eqeqeq": true,
+  "strict": true,
+  "undef": true,
+  "bitwise": true,
```

Comment:
I think this is generating a chunk of the errors, but it may possibly make sense to just mute that error on a per function basis rather than globally.